src/clients/api/queries/getLegacyPool/appendPrimeSimulationDistributions.ts

```diff
@@ -19,6 +19,27 @@ export interface ResolvePrimeSimulationDistributionsInput {
   accountAddress?: string;
 }
 
+const supplyAveragesForToken: Record<string, BigNumber> = {
+  BTCB: new BigNumber('0.71'),
+  ETH: new BigNumber('9.86'),
+  USDT: new BigNumber('5003.94'),
+  USDC: new BigNumber('13068.75'),
+} as const;
+
+const borrowAveragesForToken: Record<string, BigNumber> = {
+  BTCB: new BigNumber('0.04'),
+  ETH: new BigNumber('0.49'),
+  USDT: new BigNumber('10009.21'),
+  USDC: new BigNumber('2405.43'),
+} as const;
+
+const xvsStakedAveragesForToken: Record<string, BigNumber> = {
+  BTCB: new BigNumber('4124.59'),
+  ETH: new BigNumber('4788.05'),
+  USDT: new BigNumber('3731.33'),
+  USDC: new BigNumber('3265.30'),
+} as const;
```

Comment:
You can remove `as const` for each variable: it does nothing here since you're declaring the type explicity.